pkg/kubelet/status/status_manager.go

```diff
@@ -404,9 +406,22 @@ func (m *manager) updateStatusInternal(pod *v1.Pod, status v1.PodStatus, forceUp
 	}
 
 	normalizeStatus(pod, &status)
+
 	// The intent here is to prevent concurrent updates to a pod's status from
 	// clobbering each other so the phase of a pod progresses monotonically.
-	if isCached && isPodStatusByKubeletEqual(&cachedStatus.status, &status) && !forceUpdate {
+	shouldUpdate := forceUpdate
+	if !shouldUpdate && m.lastUpdateTime != nil {
+		duration := time.Now().Sub(*m.lastUpdateTime)
+		klog.V(3).Infof("it has been %v since last update.", duration)
```

Comment:
I think show pod id in this log would be helpful for troubleshooting.

Reply:
Pod name is included in the log below on line 419